stdpopsim/drosophila_melanogaster.py

```diff
@@ -7,6 +7,7 @@
 import stdpopsim.models as models
 import stdpopsim.genomes as genomes
 import stdpopsim.genetic_maps as genetic_maps
+from stdpopsim.generic_models import * # NOQA
```

Comment:
As a style thing, I tend to avoid ``import *``. The idea is that it's nice to know the difference between a function that's defined in the current module and one that's come from somewhere else. Then, when you refer to it, *where* the function is coming from it explicit, so you know exactly where to find it when you go looking.

Reply:
I agree this is ugly. I can get around this if generic models inherit from the model class directly and then the organism specific models inherit from the generic model class as you suggest above. My goal with this inherit was to allow for calls like `homo_sapiens.generic_constant_size()` without having to write a new `import` line for each new model.

Reply:
Yeah - we could also just put the generic models code into the ``models.py`` file --- probably overkill to have a separate module since, as you say, a lot of concrete models will be inheriting directly from these (so, we'd have models.ConstantSize, etc).

Reply:
what about this proposal: we keep the `generic_models.py` module and then have each organism `import generic_models as generics` such that one could call `homo_sapiens.generics.GenericConstantSize()`? 

That's sort of a middle of the road solution.